src/main/java/com/example/onlinebookstore/dto/BookDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.onlinebookstore.dto;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
It's not an entity, please remove annotations
```suggestion
public class BookDto {
    private Long id;
```